app/controllers/users/registrations_controller.rb

```diff
@@ -3,7 +3,9 @@
 class Users::RegistrationsController < Devise::RegistrationsController
   prepend_before_action :check_captcha, only: [:create]
   before_action :configure_sign_up_params, only: [:create]
-  invisible_captcha only: %i[create update], honeypot: :subtitle unless Rails.env.test?
+  before_action :configure_account_update_params, only: [:update]
+  respond_to :html, :js
+  invisible_captcha only: %i[create], honeypot: :subtitle unless Rails.env.test?
```

Comment:
Related ?

Reply:
Yeah, accepting GDPR policy is basically an update operation to the user model. Also, the invisible captcha is not a related change but I don't think we are having captcha div in the update page; so disabled it at the controller level too.